common-ui-view/src/main/java/app/tivi/ui/SnackbarManager.kt

```diff
@@ -33,6 +36,23 @@ class SnackbarManager @Inject constructor() {
     private val pendingErrors = MutableSharedFlow<UiError>(extraBufferCapacity = 3)
     private val removeErrorSignal = MutableSharedFlow<Unit>(extraBufferCapacity = 1)
 
+    val flow: Flow<UiError?> = flow {
```

Comment:
Can you add comments about what's the purpose of this?

Reply:
Added.